subprojects/ide-native/src/main/java/org/gradle/ide/xcode/tasks/GenerateXcodeWorkspaceFileTask.java

```diff
@@ -32,12 +32,16 @@
  */
 @Incubating
 public class GenerateXcodeWorkspaceFileTask extends XmlGeneratorTask<XcodeWorkspaceFile> {
-    private DefaultXcodeWorkspace xcodeWorkspace;
+    private Provider<Iterable<File>> xcodeProjectLocations;
```

Comment:
This should be a `private final ConfigurableFileCollection` like elsewhere, into which the `Provider` can be added.